.github/workflows/build.yml

```diff
@@ -0,0 +1,172 @@
+name: Build
+
+on:
+  push:
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build-web:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        name: Clone
+      - uses: actions/setup-node@v1
+        name: Setup Node
+        with:
+          node-version: "*"
+      - name: Install Yarn
+        run: corepack enable
+      - name: Install npm dependencies
+        run: yarn install
+      - name: Build
+        run: yarn run build
+      - uses: actions/upload-artifact@v3
+        name: Upload Artifacts
+        with:
+          name: tauri-web
+          path: dist/*
+  build-linux:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        name: Clone
+      - uses: actions-rs/toolchain@v1
+        name: Setup Rust
+        with:
+          toolchain: stable
+          override: true
+      - uses: actions/setup-node@v1
+        name: Setup Node
+        with:
+          node-version: "*"
+      - name: Install Yarn
+        run: corepack enable
+      - name: Install npm dependencies
+        run: yarn install
+      - name: Install tauri dependencies
+        run: sudo apt update && sudo apt install -y build-essential libpython3-dev libdbus-1-dev libgtk-3-dev libsoup-gnome2.4-dev libwebkit2gtk-4.0-dev
```

Comment:
Is `build-essential` really necessary? Also stylistically, I would prefer a list of commands, i.e.,

```yaml
run: |
  sudo apt update
  sudo apt install -y # ...
```

I don't know if it's any good, but `awalsh128/cache-apt-pkgs-action` may be a better approach (the list of packages installed here is long, especially considering the transitive dependencies)

Reply:
I will look into it